model/finance.php

```diff
@@ -460,4 +461,28 @@ function zustatek() {
     return $this->zustatek;
   }
 
+  /**
+   * Zjistí datum posledního provedení platby
+   */
+  private function posledniPlatba(){
+    $uid = $this->u->id();
+    $o = dbQuery("
```

Comment:
Mrkni na jiné volání dbQuery, kde se používá druhý argument s polem proměnných, na které potom v dotazu odkazuješ. Je to bezpečnější a preferovaná varianta.

```
... = dbQuery('SELECT bla bla WHERE neco = $1', [$promennaCoChciJako$1, ...]);
```

Pro tento případ to pak ještě zjednodušíš funkcí `dbOneCol`, která funguje prakticky stejně, jen ti už vrátí konkrétní jedinou hodnotu (tzn. nemusíš ručně řešit fetch a tyto věci kolem).

```
$posledniPlatba = dbOneCol('SELECT max(provedeno) FROM ...